src/main/java/com/csforge/sstable/Query.java

```diff
@@ -0,0 +1,178 @@
+package com.csforge.sstable;
+
+import com.csforge.sstable.reader.CassandraReader;
+import com.csforge.sstable.reader.Partition;
+import org.apache.cassandra.config.*;
+import org.apache.cassandra.cql3.*;
+import org.apache.cassandra.cql3.restrictions.StatementRestrictions;
+import org.apache.cassandra.cql3.selection.Selection;
+import org.apache.cassandra.cql3.statements.*;
+import org.apache.cassandra.db.*;
+import org.apache.cassandra.db.filter.ClusteringIndexFilter;
+import org.apache.cassandra.db.filter.ClusteringIndexNamesFilter;
+import org.apache.cassandra.db.filter.ClusteringIndexSliceFilter;
+import org.apache.cassandra.db.filter.ColumnFilter;
+import org.apache.cassandra.db.partitions.UnfilteredPartitionIterator;
+import org.apache.cassandra.db.rows.UnfilteredRowIterator;
+import org.apache.cassandra.dht.Murmur3Partitioner;
+import org.apache.cassandra.exceptions.InvalidRequestException;
+import org.apache.cassandra.io.sstable.Descriptor;
+import org.apache.cassandra.io.sstable.format.SSTableReader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+public class Query {
+    private static final Logger logger = LoggerFactory.getLogger(Query.class);
+
+    public static QueryOptions OPTIONS = QueryOptions.DEFAULT;
+
+    static {
+        Config.setClientMode(true);
+
+        // Partitioner is not set in client mode.
+        if (DatabaseDescriptor.getPartitioner() == null)
```

Comment:
D'oh, my bad, that's how it actually works using tableFromCQL.  So really we only need this if a schema is provided.  We could probably move this code into CassandraUtils as right now it's duplicated in Query and SSTable2Json, will fix.